source/Government.cpp

```diff
@@ -123,33 +165,18 @@ void Government::Load(const DataNode &node)
 			}
 		}
 		else if(key == "penalty for")
-		{
+			PenaltyHelper(child, penaltyFor);
+		else if(key == "custom penalties")
 			for(const DataNode &grand : child)
-				if(grand.Size() >= 2)
+			{
+				if(grand.Token(0) == "remove" && grand.Size() >= 2)
+					customPenalties[GameData::Governments().Get(grand.Token(1))->id].clear();
+				else
 				{
-					if(grand.Token(0) == "assist")
-						penaltyFor[ShipEvent::ASSIST] = grand.Value(1);
-					else if(grand.Token(0) == "disable")
-						penaltyFor[ShipEvent::DISABLE] = grand.Value(1);
-					else if(grand.Token(0) == "board")
-						penaltyFor[ShipEvent::BOARD] = grand.Value(1);
-					else if(grand.Token(0) == "capture")
-						penaltyFor[ShipEvent::CAPTURE] = grand.Value(1);
-					else if(grand.Token(0) == "destroy")
-						penaltyFor[ShipEvent::DESTROY] = grand.Value(1);
-					else if(grand.Token(0) == "scan")
-					{
-						penaltyFor[ShipEvent::SCAN_OUTFITS] = grand.Value(1);
-						penaltyFor[ShipEvent::SCAN_CARGO] = grand.Value(1);
-					}
-					else if(grand.Token(0) == "provoke")
-						penaltyFor[ShipEvent::PROVOKE] = grand.Value(1);
-					else if(grand.Token(0) == "atrocity")
-						penaltyFor[ShipEvent::ATROCITY] = grand.Value(1);
-					else
-						grand.PrintTrace("Skipping unrecognized attribute:");
+					auto &pens = customPenalties[GameData::Governments().Get(grand.Token(1))->id];
+					PenaltyHelper(grand, pens);
```

Comment:
just put pens directly into it honestly except if it goes over the max amount of lines

Reply:
It's not too long but I think it's more readable like this.